src/Api/ApiPluginInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerPlugin\Api;
+
+interface ApiPluginInterface
+{
+    /**
+     * Gets features this plugin can handle.
+     *
+     * @return array
+     */
+    public function getApiFeatures(): array;
+
+    /**
+     * Allows to override features that are provided by the Contao Managed Edition.
+     *
+     * @param array $features
+     *
+     * @return array
+     */
+    public function overrideApiFeatures(array $features): array;
```

Comment:
This forces everyone to implement the method, although 99% of the developers will never do anything with it.  I see two alternatives:

1. Move the method to a separate interface (e.g. `ApiOverridePluginInterface`).

2. Pass `&$features` by reference, so we can at least omit the `return $features;` line.

Reply:
I agree I noticed this as well. Maybe an `OverrideApiFeaturePluginInterface` would do. Though one could argue that 99% don't need this interface anyway because it's currently only useful to the core …

Reply:
nothing… 😂 it _could_ be used by another plugin… but well …

Reply:
Then remove it and re-add it when we need it.

Reply:
Agreed with @leofeyer

Reply:
Removed in 15898896cd96a6fcfece1d8ad967de9200dc17c4